elastic/kibana
# [Ingest Manager] Send kibana version on requests to package registry

## Summary

Implements https://github.com/elastic/kibana/issues/64868

This change adds the parameter `kibana.version` to all requests to the package registry's `/search` endpoint. See also https://github.com/elastic/package-registry/pull/518 for the query parameter.

## How to test this

Unfortunately, there is no easy way to test this, other than verifying that the integrations in Ingest Manager still appear and are installable as expected.

To see how the new parameter affects the registry's response, visit the following links and play around with the package name and the version number in the query string.

- https://epr-snapshot.ea-web.elastic.dev/search?package=endpoint&kibana.version=8.0.0 (responds with the package)
- https://epr-snapshot.ea-web.elastic.dev/search?package=endpoint&kibana.version=7.8.0 (responds with an empty array)
- https://epr-snapshot.ea-web.elastic.dev/search?kibana.version=7.9.0 (long list)
- https://epr-snapshot.ea-web.elastic.dev/search?kibana.version=6.0.0 (much shorter list)

For example, a package with this `condition` in its `manifest.yml` will not appear when the version sent is earlier than `7.9.0`:

```
conditions:
  kibana:
    version: ">=7.9.0"
```
